medusa/server/api/v2/show.py

```diff
@@ -86,6 +90,18 @@ def _handle_detailed_show(self, tv_show, query):
                 new, existing = tv_show.get_backlogged_episodes(allowed_qualities=allowed_qualities,
                                                                 preferred_qualities=preferred_qualities)
                 data = {'new': new, 'existing': existing}
+            elif query == 'manualSearch':
+                season = self._parse(self.get_argument('season', default=None), int)
```

Comment:
I know this is just an experiment, but manual search is not a show property or sub resource. We should use normal rest query capabilities. 
Probably `/show/tvdb1234/s03/releases?a=b` when searching releases for a given season, and `/show/tvdb1234/s01e02/releases?c=d` when searching releases for a given episode